kuwala/core/backend/app/database/models/export_block.py

```diff
@@ -0,0 +1,20 @@
+from sqlalchemy import JSON, Column, ForeignKey, Numeric, String
+from sqlalchemy.dialects.postgresql import ARRAY
+from sqlalchemy.ext.mutable import MutableList
+
+from ..database import Base
+
+
+class ExportBlock(Base):
+    __tablename__ = "export_blocks"
+
+    id = Column(String, primary_key=True, index=True)
+    export_catalog_item_id = Column(
+        String, ForeignKey("export_catalog_items.id"), nullable=False
+    )
+    data_source_id = Column(String, ForeignKey("data_sources.id"), nullable=False)
+    input_block_ids = Column(ARRAY(String))
```

Comment:
Also this shouldn't be nullable

Reply:
resolved, set `input_block_ids` to not nullable.